### ISSUE TITLE
representation of a nuc conversion do not point a conversion on the genome

### ISSUE THREAD

**Issue by huzuner** [nextstrain/ncov#1107]
Hello,

For our research, I have been using resources from Nextstrain and I encountered an ambiguity that I wanted to ask about.

I am currently working on the tabular file for nuc conversions belonging to Nextstrain clades (https://github.com/nextstrain/ncov/blob/master/defaults/clades.tsv). I have been assuming that the clades.tsv represents nucleotide conversions, so they should inform about a nucleotide change compared to the reference genome. 
However, I noticed that one of the conversions listed there, 22F nuc 15461 A, already points to A on the reference genome (NC_045512v2). Does this have an intended purpose or do I misunderstand the concept?

Thank you in advance for you response.

**Comment by jameshadfield** [comment]
Yes it looks like [there are no (or very few) mutations observed at this position](https://next.nextstrain.org/ncov/gisaid/global/6m?c=gt-nuc_15461&gmax=16461&gmin=14461). It may have been a typo. Using all 4 defining positions we can see how they identify [22F](https://next.nextstrain.org/ncov/gisaid/global/6m?c=gt-nuc_15461,17859,23019,26275&gmax=27275&gmin=14461). 

Note that these definitions are the ones we use for `augur clades` to define clades on a tree. They're not intended to be used for individual sequence classification. We recommend using [nextclade](https://clades.nextstrain.org) for classification of sequences.

**Comment by huzuner** [comment]
Thanks for your answer!

What is the difference between augur clades and nextstrain clades? Are they not same or similar?

In my case, I need either full fasta sequences or nucleotide changes of Nextstrain/Nextclade clades so that I can create 
fasta sequences myself. I did not find any of this in 'Nextclade datasets'. The only resource that is relevant is 'tree.json' but we identified some inconsistencies for some clades when compared with with covariants.org.
That's how I ended up with this 'clades.tsv', assuming that it provides the actual nucleotide differences belonging to clades.

**Comment by jameshadfield** [comment]
cc @corneliusroemer - I think you have a fasta of representative sequences for each lineage?

**Comment by huzuner** [comment]
> cc @corneliusroemer - I think you have a fasta of representative sequences for each lineage?

Would be great to hear if this exists.
